Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -443,72 +443,72 @@
 workflows[136.9] = ['', ['RunDoubleMuon2016C', 'RAWRECOTE16', 'RAWRECOLHECLEANTE16', 'EMBEDHADTE16', 'EMBEDMINIAOD16']]
 
 ### run 2021 collisions ###
-workflows[139.001] = ['RunMinimumBias2021',['RunMinimumBias2021','HLTDR3_2022','RECODR3_reHLT_MinBiasOffline','HARVESTD2021MB_reHLT']]
-workflows[139.002] = ['',['RunZeroBias2021','HLTDR3_2022','RECODR3_reHLT_ZBOffline','HARVESTD2021ZB_reHLT']]
-workflows[139.003] = ['',['RunHLTPhy2021','HLTDR3_2022','RECODR3_reHLT_HLTPhysics_Offline','HARVESTD2021HLTPhy_reHLT']]
-workflows[139.004] = ['',['RunNoBPTX2021','HLTDR3_2022','RECODR3_reHLT_AlCaTkCosmics_Offline','HARVESTDR3_reHLT']]
+workflows[139.001] = ['RunMinimumBias2021',['RunMinimumBias2021','HLTDR3_2023','RECODR3_reHLT_MinBiasOffline','HARVESTD2021MB_reHLT']]
```

Comment:
Do we want to reHLT of previous year data with new HLT menu? Compare to Run-2 reHLT in the same file, we do reHLT using the menu of the same year data taking.

Reply:
Yes (at least for now). The reason is that, in 13_1_X and 13_0_X, the HLT GT used by these workflows (`run3_hlt_relval`) uses the 2023 L1T menu, so these wfs require this HLT menu. In general, I think we should have Run-3 data workflows that exercise the latest Run-3 HLT menu (as done here).